db/schema.rb

```diff
@@ -53,17 +53,19 @@
 
   create_table "preserved_copies", force: :cascade do |t|
     t.integer "version", null: false
-    t.bigint "last_audited"
     t.bigint "preserved_object_id", null: false
     t.bigint "endpoint_id", null: false
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
-    t.datetime "last_checked_on_storage"
+    t.datetime "last_moab_validation"
```

Comment:
I think it was Julian who suggested using bigint (seconds since the epoch) for this field. Maybe run this change by him to see if there was an important reason that we might have missed with the change to datetime? I do agree that it's preferably to use a proper date abstraction instead of counting seconds since some randomly chosen time in the past.

Reply:
my recollection is that it was a performance concern.  if @julianmorley would be willing, i'd like to try just using dates and seeing how that goes.  i have to imagine the postgres date indexing is at least somewhat performant?  seems worth testing?  i agree, a proper date abstraction would be nice.  and while it didn't seem terrible to keep a shadow bigint column for one audit field, keeping a shadow column for multiple audit timestamps starts to feel (to me) like too much overhead (coginitive and storage), when we could just hopefully have our db do that for us.  and a single `last_audited` column that's the last of any of those feels less useful because presumably we want to query at the level of granularity we're tracking.

i was going to suggest that we need to start indexing those date columns, but looks like that's already part of #439!  maybe we should also index some/all of the `updated_at` fields in the schema?

Reply:
I am glad you are calling this out here.

I suspect julian wanted bigint to save a byte or two per record.  However, the bigint approach only goes to the granularity of seconds and was making us have to "sleep" for tests involving that field. 

I have grown dubious of Julian's performance concerns and went in the direction of what made the most sense cognitively and writing code wise.

Reply:
Ensure with Julian that performance is adequate.   This has more to do with date query performance than storage costs, I imagine.  We believe postgres can do date searching performantly.

Reply:
Julian has confirmed that datetime is fine by him.